bin/test-cleanup

```diff
@@ -24,6 +24,14 @@ if ! clusterroles=$(kubectl --context=$k8s_context get clusterroles -oname | gre
   echo "no clusterroles found for [$linkerd_namespace]" >&2
 fi
 
-if [[ $namespaces || $clusterrolebindings || $clusterroles ]]; then
-  kubectl --context=$k8s_context delete --wait=false $namespaces $clusterrolebindings $clusterroles
+if ! webhookconfigs=$(kubectl --context=$k8s_context get mutatingwebhookconfigurations -oname | grep -E  "/linkerd-$linkerd_namespace(-|$)"); then
+  echo "no mutatingwebhookconfigurations found for [$linkerd_namespace]" >&2
+fi
+
+if ! validatingconfigs=$(kubectl --context=$k8s_context get validatingwebhookconfigurations -oname | grep -E  "/linkerd-$linkerd_namespace(-|$)"); then
+  echo "no validatingwebhookconfigurations found for [$linkerd_namespace]" >&2
+fi
+
+if [[ $namespaces || $clusterrolebindings || $clusterroles || $webhookconfigs || $validatingconfigs ]]; then
```

Comment:
This can be replaced with `kubectl delete --ignore-not-found=false`. I didn't check when this option was introduced though. Mine is v1.13.0.